play/jaffle-shop-dbt/README.md

```diff
@@ -0,0 +1,149 @@
+## dbt's `jaffle_shop` + Materialize
+
+If you've used dbt, odds are that you've run across dbt's beloved [`jaffle_shop`] demo project.
+`jaffle_shop` allows users to quickly get up and running with dbt, using some spoofed, static
+data for a fictional [jaffle shop].
+
+At [Materialize], we specialize in maintaining fast and efficient views over your streaming data.
+While we work on hosting a public source of demo streaming data for analytics, we wanted to provide
+those familiar with dbt with an easy way to get up and running with our [`dbt-materialize`] adapter and
+`jaffle_shop`'s static data.
+
+> Note: This demo won't highlight what's powerful about Materialize. For that, check out our
+  [`wikirecent-dbt` demo] or [our docs]!
+
+[`jaffle_shop`]: https://github.com/fishtown-analytics/jaffle_shop
+[jaffle shop]: https://australianfoodtimeline.com.au/jaffle-craze/
+[Materialize]: https://materialize.com/
+[`dbt-materialize`]: https://pypi.org/project/dbt-materialize/
+[`wikirecent-dbt` demo]: https://github.com/MaterializeInc/materialize/blob/main/play/wikirecent-dbt/README.md
+[our docs]: https://materialize.com/docs/
+
+## Setting up a `jaffle_shop` with Materialize
+
+Setting up the `jaffle_shop` project with Materialize is similar to setting it up with any other
+data warehouse. The following instructions are based off the [traditional `jaffle_shop`] steps with a few
+Materialize-specific modifications:
+
+1. Following [the usual `jaffle_shop` instructions], install dbt, clone the `jaffle_shop` repository, and navigate
+   to the cloned repo on your machine.
+
+1. In your cloned `dbt_project.yml`, make the following changes to the [model materializations]:
+    ```nofmt
+    models:
+        jaffle_shop:
+            marts:
+                core:
+                    materialized: materializedview
+                    intermediate:
+                        materialized: view
+            staging:
+                materialized: view
+                tags: ["staging", "hourly"]
+    ```
+    Tip: Only materializing your `core` business models as materialized views, without materializing
+    your intermediate or staging views, ensures that you're only using the memory you need in
```

Comment:
Great point! We actually don't have a great resource for more information at the moment... putting it on our todo list.

Reply:
Awesome! :) I suspect users will be interested in knowing what this does to their performance, costs, etc, especially since most folks are used to defaulting to views instead.